development/includes/_top-nav.pug

```diff
@@ -0,0 +1,15 @@
+nav(class="navbar navbar-default navbar-fixed-top")
+  div.container-fluid
+    div.navbar-header
```

Comment:
Delete `div.container-fluid` and `div.navbar-header` they serve no real purpose in the mark up. You're wrapping your elements with extra divs for now reason.